packages/view/backend/sirius-components-view-emf/src/test/java/org/eclipse/sirius/components/view/emf/view/form/FormIdProviderTests.java

```diff
@@ -84,7 +84,7 @@ private FormDescription createFormInsideResource() {
      *
      * @author Arthur Daussy
      */
-    private class MockedObjectService extends IObjectService.NoOp {
+    private final class MockedObjectService extends IObjectService.NoOp {
```

Comment:
Checkstyle shows an error that final modifer should be added

Reply:
It can even be static on top of that, I'll had that.